XRTK-Core/Packages/com.xrtk.core/Editor/Profiles/MixedRealityToolkitRootProfileInspector.cs

```diff
@@ -88,7 +89,7 @@ protected override void OnEnable()
             var prefabStage = PrefabStageUtility.GetCurrentPrefabStage();
 
             // Create The MR Manager if none exists.
-            if (!MixedRealityToolkit.IsInitialized && prefabStage == null)
+            if (!MixedRealityToolkit.IsInitialized && prefabStage == null && !didPromptToConfigure)
```

Comment:
I think having this as an editor pref might be better

Reply:
Hmm not sure how to do that. If I save it to editor prefs "Later" will mean "Never again" unless we implement some logic to clear the value after time x has passed.

Reply:
Nevermind. I can store a timestamp instead of a bool obviously

Reply:
I mean don't reinvent the wheel, we're already doing something similar in `MixedRealityEditorSettings`

Reply:
I think this works as is for now. I think the suggestion was overboard.